actions/health-check/action.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Run MPyL health check
+description: Perform MPyL health check inside a Docker container
+inputs:
+  config-path:
+    description: The path to the MPyL config file
+    required: false
+runs:
+  using: docker
+  image: docker://public.ecr.aws/vdb-public/gh-mpyl:v0.3.0
```

Comment:
We'll have to manually maintain this version in all these actions? Meh

Reply:
No, [it's updated automatically during the release workflow](https://github.com/Vandebron/gh-mpyl/blob/main/.github/workflows/ci-release.yaml#L81-L93). Here's an example: https://github.com/Vandebron/gh-mpyl/commit/27da3204db6d283b140903263c6d64b42cd56c37